products/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+# Create your models here.
+
+class Category(models.Model):
+    name              = models.CharField(max_length=40)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name         = models.CharField(max_length=40)
+    category     = models.ForeignKey('Category',on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    name         = models.CharField(max_length=40)
+    price        = models.DecimalField(max_digits=8, decimal_places=2)
+    weight       = models.CharField(max_length=10)
+    sub_name     = models.CharField(max_length=40)
+    descriptio   = models.TextField()
+    sub_category = models.ForeignKey('SubCategory',on_delete=models.CASCADE)
+    tag          = models.ManyToManyField('Tag', through='ProductTag')
+    
+    class Meta:
+        db_table = 'products'
+
+class Tag(models.Model):
+    name         = models.CharField(max_length=40)
+    
+    class Meta:
+        db_table = 'tags'
+
+class ProductTag(models.Model):
+    product      = models.ForeignKey('Product', on_delete=models.CASCADE)
+    tag          = models.ForeignKey('Tag', on_delete=models.CASCADE)
+   
+    class Meta:
+        db_table = 'product_tags'
+
+class Image(models.Model):
+    url         = models.CharField(max_length=200)
+    product      = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'imege'
```

Comment:
```suggestion
class ProductImage(models.Model):
    url    = models.CharField(max_length=2000)
    product = models.ForeignKey('Product', on_delete=models.CASCADE)

    class Meta:
        db_table = 'product_imeges'
```
- 이미지 테이블 경우 여러 종류의 이미지 테이블이 생길 수 있습니다. 따라서 어떤 이미지를 저장하는 테이블인지 나타내주는 것이 좋습니다.
- 이미지의 url 같은 경우 크기가 가볍게 500이 넘는 것들도 많습니다. 그래서 넉넉하게 잡아주면 좋을 것 같습니다.